src/action-parser.js

```diff
@@ -0,0 +1,23 @@
+const repoRegex = /^([A-Za-z0-9_.-]+)\/([A-Za-z0-9_.-]+)(@([A-Za-z0-9_.-]+))?$/
+const convertActionToCloneCommand = (cloneDir, action, cloneUrlBuilder) => {
+  const match = repoRegex.exec(action)
+  // Validate the format
+  if (!match) throw new Error(`The action ${action} does not match with the format org/owner(@branch|@tag)? format`)
```

Comment:
I would like to suggest changing the error message to something like this:
`The action ${action} does not match with the format (org|owner)/repo[@(branch|tag)] format`
This way will be clear that the org/name and also the repo name are required parts and the also making clear that the ref is optional. What are your thoughts?

Reply:
Agreed. Also will remove the double format word, seems a typo 👍